packages/nollup/package.json

```diff
@@ -26,12 +26,15 @@
 	"homepage": "https://github.com/JoviDeCroock/prefresh#readme",
 	"dependencies": {
 		"@prefresh/core": "^1.0.0",
-		"@prefresh/utils": "^1.0.0"
+		"@prefresh/utils": "file:../../packages/utils"
```

Comment:
Not here, do it in the `test/fixture/<x>/package.json`

Reply:
Would I need to add in devDependencies too, or just resolutions will do?

Reply:
just resolutions will do, remove these since these would affect the published packages